frontend/app/LetterForm.tsx

```diff
@@ -0,0 +1,114 @@
+"use client";
+
+import React, { useState, ChangeEvent, FormEvent } from "react";
+import { solveBestTexts, SolveRequestPayload, SolveResponse } from "./api";
+
+// Generate the alphabet letters
+const letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("");
+
+const AlphabetForm: React.FC = () => {
```

Comment:
Would it be better to name this LetterForm as in the file name?

Reply:
Good catch, renamed it to "LetterForm" for better consistency.

Also good point regarding the React.FC type. I checked the discussion on it and this [React/TSX tutorial](https://react.dev/learn/typescript) and opted to remove the type for the component completely, since it doesn't take any props.